src/utils/message.js

```diff
@@ -52,3 +52,24 @@ export const findAnchor = (
   const firstUnreadMessage = findFirstUnread(messages, flags, subscriptions, mute);
   return firstUnreadMessage ? firstUnreadMessage.id : 0;
 };
+
+export const getLinkToMessage = (auth: Auth, message: Message): string => {
+  // Display_recipient contains name of stream or an array of users.
```

Comment:
I guess better place for this is `modelTypes.js`

```
display_recipient: $FlowFixMe, // `string` for type stream, else PmRecipientUser[].
```

Reply:
Will fix. It's a trace left behind from a previous version.